site/app/controllers/forum/ForumController.php

```diff
@@ -1401,7 +1401,8 @@ public function toggleLike(): JsonResponse {
 
         $responseData = [
             'status' => $output[0], // 'like'
-            'likesCount' => $output[1] // The likes count
+            'likesCount' => $output[1], // The likes count
+            'likesFromStaff' => $output[2] // Likes from staff
```

Comment:
Indexing by magic numbers is generally a poor idea.  It would be better to use an associative array for this.